docs/DeveloperGuide.md

```diff
@@ -412,12 +412,12 @@ Priorities: MVP (must have), 2 (nice to have), 3 (unlikely to have)
 
 (For all use cases below, the **System** is the `TutorEase` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use Case: UC01 - Add student contact**
+**Use Case: UC01 - Add contact**
 
 **MSS**:
 
-1. Tutor keys in required fields to add student contact.
-2. TutorEase adds the student contact.  
+1. Tutor keys in required fields to add a contact.
+2. TutorEase adds the contact.  
```

Comment:
Since we just found out that there is a markdown convention should use 1. for all the lists?

Reply:
Ok sure!